src/platform/nrfconnect/OTAImageProcessorImpl.cpp

```diff
@@ -125,5 +126,43 @@ CHIP_ERROR OTAImageProcessorImpl::ProcessHeader(ByteSpan & block)
     return CHIP_NO_ERROR;
 }
 
+// external flash optimization
+void ExtFlashHandler::DoAction(Action action)
+{
+    // introduce the QSPI driver into idle power mode
+    static constexpr auto sFlashDevice{ DT_LABEL(DT_INST(0, nordic_qspi_nor)) };
+    const auto * qspi_dev = device_get_binding(sFlashDevice);
```

Comment:
I think the current trend is to use `DEVICE_DT_GET` macro. Would sth like that work?

```suggestion
    const auto * qspi_dev = DEVICE_DT_GET(DT_NODE_LABEL(nordic_qspi_nor));
```

Reply:
If we want to use the `nordic_qspi_nor` *compatible* from .dts the `DT_INST` is the way to go here, BUT it is not recommended according to the spec:
https://docs.zephyrproject.org/latest/reference/devicetree/api.html#c.DT_INST

@Damian-Nordic seems like the proper usage of `DEVICE_DT_GET` macro in this case would be:

`const auto * qspi_dev = DEVICE_DT_GET(DT_NODELABEL(mx25r64));`

So we would need to use the exact external flash *label* from .dts which is `mx25r64`. Still better than the aforementioned non-recommended `DT_INST` imho.